README.md

```diff
@@ -100,9 +99,7 @@ git commit -a                           # Make a new commit and automatically "a
 git commit -am "I still do!"            # A combination of the above
 git commit --amend                      # Re-do the commit message of the previous commit (don't do this after pushing!)
                                         #   We _never_ change "public history"
-
-git reset <commit_hash>                 # Undo all commits after <commit_hash> (local changes are preserved)
-git reset --soft <file>                 # Unstage a staged file leaving in working directory without losing any changes. 
+git reset --soft [commit_hash]          # Unstage a staged file leaving in working directory without losing any changes. 
```

Comment:
The command and the description do not match.

Either it is:
`git reset <file>   # Unstage a staged file leaving in working directory without losing any changes.`

or:
`git reset --soft [commit_hash]     # resets the current branch to <commit>. Does not touch the staging area or the working tree at all.`

Maybe both?

Reply:
I added both, sorry for the ping-pong.